bc_gym_planning_env/envs/base/env.py

```diff
@@ -16,7 +16,8 @@
 from bc_gym_planning_env.utilities.path_tools import refine_path
 from bc_gym_planning_env.envs.base.draw import draw_environment
 from bc_gym_planning_env.envs.base.obs import Observation
-from bc_gym_planning_env.envs.base import spaces
```

Comment:
Why not just use gym.spaces? The RL framework also uses this class. If you prefer to keep this, I'll change the one in the RL framework to from bc_gym_planning_env.envs.base import spaces

Reply:
This is because `gym.spaces` does almost nothing, and depends on `scipy`

Reply:
ok, no prob